### DISCUSSION TITLE
not an exact search

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4875] [General]
Guys, please tell me. When searching for the necessary data in the Wiki, when filling in the search bar. In addition to the necessary data that are being searched for, there are also extra data that were not searched for. Why is this happening and how to fix it?

**Comment by tommoor** [comment]
Could you provide more details and examples?

**Comment by MaksGill** [comment]
The author of the articles for which the search returned results was the same person. Some search results don't even have a connection to the search query.
![Wiki](https://user-images.githubusercontent.com/122286411/219308941-cfb04a82-82f9-4792-bb42-80d8676dcbc2.jpg)

**Reply by tommoor** [reply]
If you need an exact search you can use quotes around the search term, otherwise stemming is used and may return documents with similar spellings
